concordia/.coveralls.yml

```diff
@@ -0,0 +1,2 @@
+service_name: travis-pro
+repo_token: iqMsJqSabqMfzGoCim0foxh4VQZHKty3G
```

Comment:
I don't think `repo_token` is supposed to be public:

> Note: Do not make your repo token public. 

https://coveralls.io/github/LibraryOfCongress/concordia

Reply:
Oops, it's public for bagit-python

Reply:
https://github.com/LibraryOfCongress/bagit-python/blob/master/.coveralls.yml

Reply:
I think that comment only applies to private repos, but I dunno.

Reply:
Is there a way to regenerate that token?

Reply:
There must be

Reply:
https://coveralls.io/github/LibraryOfCongress/concordia/settings has the button to regenerate the token

Reply:
https://github.com/LibraryOfCongress/bagit-python/commit/979c1ee78fdf9957e5eb44e6473ec7f192ccd18f just confirmed that we don't need this file at all for open-source projects. See https://coveralls.io/github/LibraryOfCongress/bagit-python